tests/acceptance/features/bootstrap/GuestsContext.php

```diff
@@ -657,4 +658,38 @@ public function userHasSharedFolderWithGuestUser(
 			"User $sharer failed to share $filePath with user $guestUser"
 		);
 	}
+
+	/**
+	 * @Given the administrator has removed the app :app from the whitelist for the guest user
+	 *
+	 * @param string $app
+	 *
+	 * @return void
+	 */
+	public function theAdministratorHasRemovedAppFromTheWhitelistForGuestUser(string $app) : void {
+		$whiteList = SetupHelper::runOcc(
+			['config:app:get', 'guests', 'whitelist'],
+			$this->featureContext->getStepLineRef()
+		)['stdOut'];
+
+		$whiteList = explode(",", trim($whiteList));
+		$whiteList = array_filter(
+			$whiteList,
+			function ($item) use ($app) {
+				return $item !== $app;
+			}
+		);
+		$whiteList = join(",", $whiteList);
+		AppConfigHelper::modifyAppConfig(
+			$this->featureContext->getBaseUrl(),
+			$this->featureContext->getAdminUsername(),
+			$this->featureContext->getAdminPassword(),
+			"guests",
+			"whitelist",
+			$whiteList,
+			$this->featureContext->getStepLineRef(),
+			$this->featureContext->getOcsApiVersion()
+		);
+		$this->featureContext->clearStatusCodeArrays();
+	}
```

Comment:
I am seeing this here. And we can re-use those helper functions later
```suggestion
        public function getWhiteListApps() {
                return SetupHelper::runOcc(
			['config:app:get', 'guests', 'whitelist'],
			$this->featureContext->getStepLineRef()
		)['stdOut'];
        }
        ...
        public function removeAppFromWhiteList(string $app) {
               $whiteList = $this->getWhiteListApps();
                $whiteList = explode(",", trim($whiteList));
		$whiteList = array_filter(
			$whiteList,
			function ($item) use ($app) {
				return $item !== $app;
			}
		);
		$whiteList = join(",", $whiteList);
		AppConfigHelper::modifyAppConfig(
			$this->featureContext->getBaseUrl(),
			$this->featureContext->getAdminUsername(),
			$this->featureContext->getAdminPassword(),
			"guests",
			"whitelist",
			$whiteList,
			$this->featureContext->getStepLineRef(),
			$this->featureContext->getOcsApiVersion()
		);
        }
	public function theAdministratorHasRemovedAppFromTheWhitelistForGuestUser(string $app) : void {
		$this->removeAppFromWhiteList($app);
	}
```